src/odemis/acq/stream/_sync.py

```diff
@@ -1107,7 +1107,8 @@ def _runAcquisitionEbeam(self, future):
 
                     # integrate/sum images
                     self._integrateImages(integration_count)
-
+                    # Live update the setting stream with the new data
+                    self._sccd._onNewData(self._ccd_df, self._acq_data[self._ccd_idx][-1])
```

Comment:
Please add similar code to the `_runAcquisitionScanStage()`. Somewhere just before the `break` (l 1595) should be equivalent. As the two functions are very similar, it should work well too.